srcts/src/components/taskButton.ts

```diff
@@ -0,0 +1,73 @@
+import { InputBinding, registerBinding } from "./_utils";
+import type { BslibSwitchInline } from "./webcomponents/switch";
+
+type TaskButtonMessage = {
+  state: string;
+};
+
+/**
+ * This is a Shiny input binding for `bslib::task_button()`. It is not a web
```

Comment:
```suggestion
 * This is a Shiny input binding for `bslib::input_task_button()`. It is not a web
```

Reply:
Fixed locally so I could include the built JS in the same commit. Thanks!